json/src/lib.rs

```diff
@@ -845,6 +845,51 @@ pub struct TestMempoolAcceptResult {
     pub fees: Option<TestMempoolAcceptResultFees>,
 }
 
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
+pub struct SubmitPackageResult {
```

Comment:
I think most of the types in this struct try to follow the "declare before use" principle, so could you perhaps move the types in opposite order?